main/include/OpenThreadConfig.h

```diff
@@ -25,13 +25,18 @@
 #ifndef OPENTHREAD_PLATFORM_CONFIG_H
 #define OPENTHREAD_PLATFORM_CONFIG_H
 
+#include "ble_config.h"
+
 // Disable the Nordic-supplied OpenThread logging facilities and use
 // the facilities provided by the OpenWeave Device Layer (see
 // openweave/src/adaptations/device-layer/nRF5/Logging.cpp).
 #define OPENTHREAD_CONFIG_LOG_OUTPUT OPENTHREAD_CONFIG_LOG_OUTPUT_APP
 
 // Turn on a moderate level of logging in OpenThread
-#define OPENTHREAD_CONFIG_LOG_LEVEL OT_LOG_LEVEL_NOTE
+#define OPENTHREAD_CONFIG_LOG_LEVEL OT_LOG_LEVEL_DEBG
```

Comment:
I can revert this change.  Perhaps we can add a simple build flag that enables full logging.

Reply:
I agree.  Since the primary role of the example apps is to demonstrate Weave I want to keep the OT logging at a moderate level.  Developers can fork the example app and make their own adjustments as needed.

Reply:
Reverted to NOTE.